slides/ml_concepts.md

```diff
@@ -49,7 +47,7 @@ Some examples of machine learning
 
 ???
 
-Why and when would we use machine learning?
+First of all, I would like to discuss : Why and when would we use machine learning?
```

Comment:
I usually try to keep the sentences on slides very short

Reply:
Oops, sorry, this is not on the slide, but in the text read. I'll remove the comment above.